extensions/wikia/ArticleVideo/bower_components/html5-skin/js/views/contentScreen.js

```diff
@@ -0,0 +1,92 @@
+var React = require('react'),
+    CloseButton = require('../components/closeButton'),
+    Utils = require('../components/utils'),
+    CONSTANTS = require('../constants/constants'),
+    Icon = require('../components/icon'),
+    Watermark = require('../components/watermark'),
+    AccessibilityMixin = require('../mixins/accessibilityMixin');
+
+var ContentScreen = React.createClass({
+  mixins: [AccessibilityMixin],
+
+  handleClose: function() {
+    switch(this.props.screen) {
+      case CONSTANTS.SCREEN.DISCOVERY_SCREEN:
+        this.props.controller.toggleDiscoveryScreen();
+        break;
+      default:
+        this.props.controller.toggleScreen(this.props.screen);
+    }
+  },
+
+  render: function() {
+    //overlay only for the closed captions screen. Needs to be different than the other screens because of closed caption preview.
+    var closedCaptionOverlay = this.props.screen == CONSTANTS.SCREEN.CLOSEDCAPTION_SCREEN ? (
+      <div className="oo-closed-caption-overlay"></div>
+    ) :
+    null;
+
+    var titleBarStyle = {};
+    switch (this.props.screen) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1301)